### ISSUE TITLE
Error Code 3

### ISSUE THREAD

**Issue by YogiOnBioinformatics** [ncbi/sra-tools#348]
Hello, 

I am using `fasterq-dump` and getting this funky error when running my commands. 

`spots read      : 14,483,135
reads read      : 14,483,135
reads written   : 14,483,135
spots read      : 17,269,995
reads read      : 17,269,995
reads written   : 17,269,995
2020-06-03T22:37:42 fasterq-dump.2.10.7 err: cmn_iter.c cmn_read_String( #40251393 ).VCursorCellDataDirect() -> RC(rcXF,rcFunction,rcSelecting,rcData,rcCorrupt)
2020-06-03T22:37:42 fasterq-dump.2.10.7 err: sorter.c write_to_store().pack_read_2_4na() failed RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T22:37:42 fasterq-dump.2.10.7 err: sorter.c run_producer_pool().join_and_release_threads -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T22:37:42 fasterq-dump.2.10.7 err: sorter.c execute_lookup_production() -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T22:37:42 fasterq-dump.2.10.7 err: fasterq-dump.c produce_lookup_files() -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
fasterq-dump quit with error code 3
spots read      : 10,949,659
reads read      : 10,949,659
reads written   : 10,949,659
2020-06-03T22:48:13 fasterq-dump.2.10.7 err: sorter.c write_to_store().pack_read_2_4na() failed RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T22:48:13 fasterq-dump.2.10.7 err: sorter.c run_producer_pool().join_and_release_threads -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T22:48:13 fasterq-dump.2.10.7 err: sorter.c execute_lookup_production() -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T22:48:13 fasterq-dump.2.10.7 err: fasterq-dump.c produce_lookup_files() -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
fasterq-dump quit with error code 3
spots read      : 22,256,931
reads read      : 22,256,931
reads written   : 22,256,931
spots read      : 18,136,474
reads read      : 18,136,474
reads written   : 18,136,474
2020-06-03T23:11:21 fasterq-dump.2.10.7 err: sorter.c write_to_store().pack_read_2_4na() failed RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T23:11:21 fasterq-dump.2.10.7 err: sorter.c run_producer_pool().join_and_release_threads -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T23:11:21 fasterq-dump.2.10.7 err: sorter.c execute_lookup_production() -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
2020-06-03T23:11:21 fasterq-dump.2.10.7 err: fasterq-dump.c produce_lookup_files() -> RC(rcVDB,rcNoTarg,rcWriting,rcFormat,rcNull)
fasterq-dump quit with error code 3
`

I am using SRA-toolkit version 2.10.7 and these were the **EXACT** commands I ran that gave me the output above: 

`/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985464
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985466
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985467
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985470
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985471
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985472
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985474
/usr/local/sratoolkit/bin/fasterq-dump -O raw_fastq/ SRR10985476`

**Comment by YogiOnBioinformatics** [comment]
sorry just wanted to follow up on this.

**Comment by kwrodarmer** [comment]
Sorry for the delay in responding. Yes, we received your report and will get back to you as soon as we can.

**Comment by kwrodarmer** [comment]
You should know we have been able to duplicate the problem and are analyzing it. We'll be off over the weekend, of course, but will pick back up on Monday.

**Comment by YogiOnBioinformatics** [comment]
sounds good. 
I really appreicate it 😄

**Comment by YogiOnBioinformatics** [comment]
super sorry to bother. 
do you have any update on this?

**Comment by YogiOnBioinformatics** [comment]
oh alright, thanks so much for the update. 😄 

do you know approximately how long it'll be till that update?

**Comment by kwrodarmer** [comment]
We have discovered the issue, and are putting the changes through testing now. It should be in our code repository yet this week. If you are able to build from source, that would give you the earliest access. If not, we'll get it out within a week if all goes well.

**Comment by YogiOnBioinformatics** [comment]
alright sounds great!!!

**Comment by YogiOnBioinformatics** [comment]
@kwrodarmer 

super sorry, don't mean to bother. 
just wanted to see if the update has been pushed. 

just asking since I'm waiting to download this data and analyze it.

**Comment by kwrodarmer** [comment]
No bother at all - sorry we haven't updated this here. I expect we will get you started tomorrow or Wednesday, depending upon your needs.

We need to release this to the general public, but if you'd be willing to try a pre-release, we should be able to get that to you tomorrow. The only thing is that pre-release is just waiting on more testing before we release it generally. Let us know.

**Comment by YogiOnBioinformatics** [comment]
no worries at all! 

i can wait for the regular update if it's anyways going to be this week. 

i really appreciate it :)

**Comment by YogiOnBioinformatics** [comment]
no clue how or why but this it worked all of a sudden when re-trying it many days later.

**Comment by priyatamapandey** [comment]
Hi,
I want to follow up on the same issue. Got the same error for one of the file. How to resolve this issue?

Thank you,
Priya

**Comment by kwrodarmer** [comment]
According to @YogiOnBioinformatics , it was resolved by updating the toolkit.

Can you give any specifics, and please create a new issue since this one is closed.
Thank you.
